test/categorical_algebra/CSetDataStructures.jl

```diff
@@ -55,9 +53,9 @@ for dds_maker in dds_makers
 
   @test has_subpart(dds, :Φ)
   @test !has_subpart(dds, :nonsubpart)
-  # @test_throws ArgumentError subpart(dds, 1, :nonsubpart)
-  # @test_throws ArgumentError incident(dds, 1, :nonsuppart)
-  # @test_throws ArgumentError set_subpart!(dds, 1, :nonsubpart, 1)
+  @test_throws Any subpart(dds, 1, :nonsubpart)
```

Comment:
These still throw an except, right? If you don't want to be specific, you should use the abstract type `Exception` instead of `Any`.